### ISSUE TITLE
Edit imported REG file only?

### ISSUE THREAD

**Issue by dardoor** [Fleex255/PolicyPlus#66]
I want to import an existing REG file, edit it and export it, to create specific policies for another computer.
But when I import a REG file, Policy Plus also shows the local (current computer) policies.

So the only way to do what I want is to use POL files through "Open Policy Resources", correct?
This works so it's not a big deal, but it would be nice if it could be done with REG files too.

**Comment by Fleex255** [comment]
Right, importing a REG *adds* its settings to your current policy source. Policy Plus currently does not support REG files as policy sources, so you would have to create a blank POL and work with that, but adding support for REG sources sounds feasible.
